### ISSUE TITLE
Option to disable shape overlay window

### ISSUE THREAD

**Issue by wrgrant** [jgraph/drawio-desktop#379]
Can we have an option to disable this overlay?
Or maybe disable it by default?

I'm not sure what the point of it is, since you don't seem to be able to edit the text?
It just duplicates the view of the shape you currently have selected.

![2020-09-15_16h43_53](https://user-images.githubusercontent.com/3092780/93262915-0414e780-f773-11ea-9dfd-7dc1f2be62cb.png)

**Comment by mararad** [comment]
Hi,

Could you tell me diagrams.net version you are using (can be seen under Help) and attach the file here so I could test it?

Thanks,

**Comment by wrgrant** [comment]
13.6.2


[Untitled Diagram.zip](https://github.com/jgraph/drawio-desktop/files/5233019/Untitled.Diagram.zip)

**Comment by mararad** [comment]
I couldn't reproduce this. 

What operating system are you using? 
Did you add borders maybe? 
Did you edit textbox in any way?
Did you download a desktop app from here: https://github.com/jgraph/drawio-desktop/releases
If you uninstall and reinstall the app, do you get the same result?

Thanks,

**Comment by wrgrant** [comment]
* OS: Windows 10 1909
* If I toggle borders in Format Panel > Text > Border color, no change. (Couldn't find different borders option)
* As in editing the text in the textbox? No. Overlay shows up even if I delete all objects from the drawing.
* I downloaded it from get.drawings.net, But I just tried uninstalling, then installing that one from github and same issue.
* No layers.

I actually just seemingly fixed it though.
Solution was to:
* Close program
* Delete the directory: %appdata%\Roaming\draw.io
* Restart program

I'll close and reopen if this happens again. Maybe next time I can use debugtron or something to figure out exactly what element is being drawn. I can't figure out how to make the overlay come back up now though.

Thanks!
